lib/LANraragi/Plugin/Metadata/EHentai.pm

```diff
@@ -140,6 +150,11 @@ sub lookup_gallery {
       . "?advsearch=1&f_sname=on&f_stags=on&f_sdt2=on&f_spf=&f_spt=&f_sfu=on&f_sft=on&f_sfl=on"
       . "&f_search=" . uri_escape_utf8(qw(").$title.qw("));
 
+    #Include expunged galleries in the search if the option is enabled.
+    if ( $expunged ) {
+        $URL = $URL . "&f_sh=on";
```

Comment:
My e-h search-fu is a bit rusty, expunged really can use both `f_sh=on` and `fs_exp=1` as query parameters?

Reply:
Yeah, image search and title search use different parameters.

Reply:
dangit tenboro